src/java/htsjdk/samtools/cram/CRAIIndex.java

```diff
@@ -0,0 +1,149 @@
+package htsjdk.samtools.cram;
+
+import htsjdk.samtools.CRAMIndexer;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.cram.structure.Slice;
+import htsjdk.samtools.seekablestream.SeekableMemoryStream;
+import htsjdk.samtools.seekablestream.SeekableStream;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * A collection of static methods to read, write and convert CRAI index.
+ */
+public class CRAIIndex {
+    public static final String CRAIIndexSuffix = ".crai";
```

Comment:
Rename the constant `CRAIIndexSuffix` to java constant styled `CRAI_INDEX_SUFFIX`.

I realize that the bam/bai constant name is also incorrect. Someday that should be deprecated and fixed, but will leave it to the other htsjdk reviewers to decide if that should happen now.